pitaya-sharp/NPitaya/src/NativeInterop.cs

```diff
@@ -3,6 +3,14 @@
 
 namespace NPitaya
 {
+    public enum MetricType : byte
+    {
+        Counter = 0,
+        Gauge,
+        Histogram,
+        Summary
+    }
+
```

Comment:
We are not using this on csharp side right? I think we could remove it

Reply:
The whole enum wasn't being used at all hehe. Remove it.